tests/render/renderer/v3/test_suite_profile_notebook_renderer.py

```diff
@@ -0,0 +1,213 @@
+import os
+from typing import Set
+
+from great_expectations import DataContext
+
+# noinspection PyProtectedMember
+from great_expectations.cli.suite import _suite_edit_workflow
+from great_expectations.core import (
+    ExpectationConfiguration,
+    ExpectationSuite,
+    ExpectationSuiteValidationResult,
+)
+from tests.profile.conftest import get_set_of_columns_and_expectations_from_suite
+from tests.render.test_util import run_notebook
+
+
+def test_notebook_execution_with_pandas_backend(
+    titanic_v013_multi_datasource_pandas_data_context_with_checkpoints_v1_with_empty_store_stats_enabled,
+):
+    """
+    To set this test up we:
+
+    - create a suite using profiling
+    - verify that no validations have happened
+    - create the suite edit notebook by hijacking the private cli method
+
+    We then:
+    - execute that notebook (Note this will raise various errors like
+    CellExecutionError if any cell in the notebook fails
+    - create a new context from disk
+    - verify that a validation has been run with our expectation suite
+    """
+    # Since we'll run the notebook, we use a context with no data docs to avoid the renderer's default
```

Comment:
Just a tiny nit - this context does have a data_docs section

Reply:
Deleted the irrelevant comment (in the next commit).